runtime/src/comm/gasnet/comm-gasnet.c

```diff
@@ -255,6 +304,9 @@ static void fork_nb_large_wrapper(fork_t* f) {
   void* f_arg;
   chpl_memcpy(&f_arg, f->arg, sizeof(void*));
 
+  // MPF - should internal to the GASNet comms layer communication
+  // like this be counted in e.g. chpl_comm_commDiagnostics.get ?
+  // If not, we probably want to make e.g. do_remote_get.
```

Comment:
(Just a comment.) Historically we've counted these. Arguably as especially the module code gets more and more abstract these counters may come to have less and less meaning at least to a user programmer, but for now it's probably best to just count everything, no matter who initiates it. I'm not sure that putting the question in an obscure comment deep in the comm layer is the right place, though. Might be better to ask on one of the mailing lists!